--- gnome-chess-0.4.0/gnome-chess.desktop.in.orig	2003-08-22 00:26:16.000000000 +0200
+++ gnome-chess-0.4.0/gnome-chess.desktop.in	2007-09-05 22:23:17.519134448 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-chess.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;BoardGame;
+Categories=GTK;GNOME;Game;BoardGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-chess
 X-GNOME-Bugzilla-Component=General
