diff -Nru gnome-chess-0.2.3/Makefile.am gnome-chess-0.2.3.new/Makefile.am
--- gnome-chess-0.2.3/Makefile.am	Mon Sep 20 06:17:42 1999
+++ gnome-chess-0.2.3.new/Makefile.am	Wed Oct 20 05:45:41 1999
@@ -4,7 +4,7 @@
 Icon_DATA = gnome-chess.png
 Mime_DATA = gnome-chess.keys gnome-chess.mime
 
-Gamesdir = $(datadir)/gnome/apps/Games
+Gamesdir = $(datadir)/applnk/Games
 Icondir = $(datadir)/pixmaps
 Mimedir = $(datadir)/mime-info
 
